setup.py

```diff
@@ -45,6 +45,10 @@
         "imageio-ffmpeg>=0.4.7",
         "scikit-image",
         "trimesh",
+        "torch",
+        "torch_geometric",
+        "lightning[extra]",
+        "libigl",
```

Comment:
How heavy are these dependencies? Do they work on all platforms?

By adding the dependencies here, everyone who wants to run *some* examples (not even this one particularly) has to install these, and this might add considerable overhead. An alternative could be to document this in the example docstring, plus a one-line command to install via pip/conda.

Reply:
Torch is not small as far as I know, I'd recommend adding instructions in the docstring of the Python module that you need to install these

Reply:
Make sure to disable this one for automated test runs then

Reply:
Yea `pytorch` is a massive library, and installing it isn't exactly as trivial as just installing `torch`, depends on if you want GPU support, what CUDA version etc. We usually have users install torch independently before installing a downstream package.

Reply:
Thanks for the review, I will remove those packages form the dependency list and add a docstring comment that states that those should be installed separately.

Reply:
Fixed